hub_test.go

```diff
@@ -390,16 +391,17 @@ func TestConcurrentHubClone(t *testing.T) {
 		},
 	}
 
-	got := make([]TestEvent, len(transport.Events()))
-	for i, event := range transport.Events() {
-		got[i] = TestEvent{
+	var got []TestEvent
+	for _, event := range transport.Events() {
+		got = append(got, TestEvent{
```

Comment:
This is a rather stylistic change that requires the accompanying ignore-rule in the linter config.

The more typical idiom in Go is to declare a slice and `append` to it in the loop. Initializing with `make` is unnecessary, perhaps tending to a premature optimization (as is [documented in the very linter rule that likely made this line get this shape](https://github.com/golangci/golangci-lint/blob/d3e36a97cdc1c68566b0a274df367a44cd766cfb/.golangci.example.yml#L172-L174)).